Letterbook.Core/Extensions/DependencyInjection.cs

```diff
@@ -32,7 +34,25 @@ public static IServiceCollection AddLetterbookCore(this IServiceCollection servi
 			.AddScoped<MappingConfigProvider>()
 			.AddSingleton<Instrumentation>()
 			.AddSingleton<IAuthorizationService, AuthorizationService>()
-			.AddSingleton<IHostSigningKeyProvider, DevelopmentHostSigningKeyProvider>();
+			.AddSingleton<IHostSigningKeyProvider, DevelopmentHostSigningKeyProvider>()
+			.AddSingleton<IContentSanitizer, HtmlSanitizer>()
+			.AddSingleton<IContentSanitizer, TextSanitizer>()
+			// TODO: XssSanitizer config
+			.AddSingleton<Xss.IHtmlSanitizer, Xss.HtmlSanitizer>(_ => new Xss.HtmlSanitizer(new Xss.HtmlSanitizerOptions
+			{
+				AllowedTags = new HashSet<string>()
+				{
+					"a", "abbr", "address", "b", "bdi", "blockquote", "caption", "cite", "code", "col", "colgroup", "dd", "del", "dl", "dt",
+					"em", "i", "ins", "kbd", "li", "mark", "ol", "p", "pre", "q", "s", "strike", "strong", "sub", "sup", "table", "tbody",
+					"td", "tfoot", "th", "thead", "time", "tr", "tt", "ul", "var", "wbr"
+				},
+				AllowedAttributes = new HashSet<string>(){"href"},
+				AllowedCssClasses = new HashSet<string>(),
+				AllowedCssProperties = new HashSet<string>(),
+				AllowedAtRules = new HashSet<CssRuleType>(),
+				AllowCssCustomProperties = false,
+				AllowDataAttributes = false
```

Comment:
This is the HTML sanitization config. It applies everywhere we're sanitizing html input. Meaning, both from local posts (and profiles, etc, eventually) created by users, and remote posts received from peers. I'm very open to input on whether this is the right set of elements to allow. Note that the empty sets mean nothing is permitted in those categories.